custom_model_runner/README.md

```diff
@@ -149,11 +149,27 @@ DRUM can also run as a prediction server. To do so, provide a server address arg
 The DRUM prediction server provides the following routes. You may provide the environment variable URL_PREFIX. Note that URLs must end with /.
 
 * Status routes:   
-A GET **URL_PREFIX/** and **URL_PREFIX/ping/** routes, which checks if the server is alive.  
+A GET **URL_PREFIX/** and **URL_PREFIX/ping/** routes, shows server status - if the server is alive.  
+Example: GET http://localhost:6789/
+
+* Health route:  
+A GET **URL_PREFIX/health/** route, shows functional health. E.g. model is loaded and functioning properly.  
+Example: GET http://localhost:6789/
+  
+* Info route:  
+A GET **URL_PREFIX/info/** route, shows information about running model (metadata, paths, predictor type, etc.).  
+Example: GET http://localhost:6789/
+
+* Statistics route:  
+A GET **URL_PREFIX/stats/** route, shows running model statistics (memory).  
+Example: GET http://localhost:6789/
+
+* Capabilities route:  
+A GET **URL_PREFIX/capabilities/** route, shows payload formats supported by running model.  
```

Comment:
if we want users to be able to create their own custom environments - the description here should be very detailed.
e.g. request schema, response schema

Reply:
But these are only DRUM endpoints. 
Maybe I just should mention they are not accessible on DR prediction server

Reply:
So what are you asking for regarding this paragraph?

Reply:
> description here should be very detailed. e.g. request schema, response schema

Reply:
So not here. This is DRUM only documentation, that goes here: https://pypi.org/project/datarobot-drum/

And you are talking about creating your own custom env.

Reply:
no, i mean here. These routes are in drum server. they should be fully documented.

Reply:
ok